app/controllers/application_controller.rb

```diff
@@ -1,2 +1,22 @@
+# 404ページと500ページ
 class ApplicationController < ActionController::Base
+  unless Rails.env.production? # development?
+    rescue_from Exception, with: :render500
+    rescue_from ActiveRecord::RecordNotFound, with: :render404
+    rescue_from ActionController::RoutingError, with: :render404
+  end
+
+  def routing_error
+    raise ActionController::RoutingError, params[:path]
+  end
+
+  private
+
+  def render404
```

Comment:
render_not_foundのした方がわかりやすいかな
```suggestion
  def render_not_found
```

Reply:
https://github.com/ok-os-job-change-team/tetsuya-twitter-clone-bootcamp/pull/36/commits/bd99bf3eda53a7ebdbcf1c9182be7a70a0f9098e で変更しました。